dali/operators/image/remap/warp_affine_params.cu

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/image/remap/warp_affine_params.h"
+#include <cuda_runtime.h>
+
+namespace dali {
+namespace {
+
+template <int ndims>
+__global__ void InvertTransformsK(WarpAffineParams<ndims> *output,
```

Comment:
```suggestion
__global__ void InvertTransformsKernel(WarpAffineParams<ndims> *output,
```
this dangling K is rather foreign in our code.

Reply:
done